src/MoBi.Presentation/Presenter/HierarchicalStructurePresenter.cs

```diff
@@ -88,7 +88,7 @@ private IEnumerable<ObjectBaseDTO> neighborsOf(NeighborhoodBuilder neighborhoodB
       /// <returns>An Id that combines the two Ids of the neighborhood</returns>
       private string createNeighborhoodId(IWithId me, IWithId myNeighbor)
       {
-         return $"{me.Id}-{myNeighbor.Id}";
+         return $"{me?.Id}-{myNeighbor?.Id}";
```

Comment:
I am not quite getting it. Why aren't we using the id of the neighborhood? 
Also two neighborhood with the same source but no target would have the same id right?